config/routes.rb

```diff
@@ -10,12 +12,16 @@
     get 'signup', to: 'devise/registrations#new'
   end
 
-  resources :deliverers, only: %i[new create edit update]
-  resources :shifts, only: %i[new create edit update]
+  resources :deliverers, only: %i[index new create edit update]
+  resources :shifts, only: %i[index new create edit update]
 
-  get '/home', to: 'pages#home'
+  get '/assignments/show', to: 'assignments#show'
```

Comment:
`#show` should be used to show a single entity. The list of entities should be under `#index`
https://www.codecademy.com/articles/standard-controller-actions

Reply:
Fixed